libcloud/backup/drivers/dimensiondata.py

```diff
@@ -1,690 +1,691 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
shrug. which line ending configuration are you using? I've got autoclrf = false in git, but the file is CLRF